examples/nxp/aot_neutron_compile.py

```diff
@@ -0,0 +1,322 @@
+        "--portable_lib",
+        required=False,
+        default=None,
+        help="Provide path to portable_lib so library. Should be in cmake-out/_portable_lib.cpython-310-x86_64-linux-gnu.so",
+    )
+    parser.add_argument(
+        "--debug", action="store_true", help="Set the logging level to debug."
+    )
+    parser.add_argument(
+        "-t",
+        "--test",
+        action="store_true",
+        required=False,
+        default=False,
+        help="Test the selected model and print the accuracy between 0 and 1.",
+    )
+    parser.add_argument(
+        "--operators_not_to_delegate",
+        required=False,
+        default=[],
+        type=str,
+        nargs="*",
+        help="List of operators not to delegate. E.g., --operators_not_to_delegate aten::convolution aten::mm",
+    )
+
+    args = parser.parse_args()
+
+    if args.debug:
+        logging.basicConfig(level=logging.DEBUG, format=FORMAT, force=True)
+
+    # 1. pick model from one of the supported lists
+    model, example_inputs, calibration_inputs = get_model_and_inputs_from_name(
+        args.model_name
+    )
+    model = model.eval()
+
+    # 2. Export the model to ATEN
+    exported_program = torch.export.export_for_training(
+        model, example_inputs, strict=True
+    )
+
+    # TODO: Add Neutron ATen Passes, once https://github.com/pytorch/executorch/pull/10579 is merged
+    #  https://github.com/pytorch/executorch/issues/10898
+
+    module = exported_program.module()
+
+    # 4. Quantize if required
+    if args.quantize:
+        if args.quantize and (not args.so_library or not args.portable_lib):
+            logging.warning(
+                "Quantization enabled without supplying path to libcustom_ops_aot_lib using --so_library and "
+                "_portable_lib.cpython* using --portable_lib CLI options. \n"
+                "This is required for running quantized models with unquantized input. The script might fail with "
+                "Runtime Exception later on."
+            )
+        if calibration_inputs is None:
+            logging.warning(
+                "No calibration inputs available, using the example inputs instead"
+            )
+            calibration_inputs = example_inputs
+        module = post_training_quantize(module, calibration_inputs)
+
+        # For quantization we need to build the quantized_ops_aot_lib.so and _portable_lib.*.so
+        # Use this CMake options
+        # -DEXECUTORCH_BUILD_KERNELS_QUANTIZED=ON
+        # -DEXECUTORCH_BUILD_KERNELS_QUANTIZED_AOT=ON
+        # and build the quantized_ops_aot_lib package
+        # Then run with --so_library <path_to_quantized_ops_aot_lib> --portable_lib <path_to_portable_lib>
+        if args.so_library is not None:
+            logging.debug(
+                f"Loading libraries: {args.so_library} and {args.portable_lib}"
+            )
+            torch.ops.load_library(args.portable_lib)
+            torch.ops.load_library(args.so_library)
```

Comment:
why do we need these? just include the python module perhaps?

Reply:
You are right (obviously) , we don't. Importing the python modules instead. 
```
import executorch.extension.pybindings.portable_lib
import executorch.kernels.quantized 
```
Thanks for the finding, it helped me to locate these python modules.